tests/test_cli/test_gui.py

```diff
@@ -16,56 +16,59 @@
 #   limitations under the License.
 #
 # ------------------------------------------------------------------------------
-
 """This test module contains the tests for the `aea gui` sub-command."""
-
 import json
 import os
-import subprocess  # nosec
 import sys
 import tempfile
-import time
 from pathlib import Path
+from unittest.case import TestCase
 
 import jsonschema
 from jsonschema import Draft4Validator
 
+import pytest
+
+from tests.common.pexpect_popen import PexpectSpawn
+
 from ..conftest import (
     AGENT_CONFIGURATION_SCHEMA,
-    CLI_LOG_OPTION,
     CONFIGURATION_SCHEMA_DIR,
     tcpping,
 )
 
 
-class TestGui:
+class TestGui(TestCase):
     """Test that the command 'aea gui' works as expected."""
 
-    @classmethod
-    def setup_class(cls):
+    def setUp(self):
```

Comment:
is `setUp` the setup method of `TestCase`?

Reply:
yes, fixed. changed to pytest's style